design/ceph/object-store.md

```diff
@@ -95,51 +97,19 @@ The RGW service can be configured to listen on both http and https by specifying
     instances: 1
 ```
 
-### Realms, Zone Groups, and Zones
-
-By default, the object store will be created independently from any other object stores and replication to another object store will not be configured. This done by creating a new realm, zone group, and zone all with the name of the new object store. The zone group and zone are tagged as the `master`. If this is the first object store in the cluster, the realm, zone group, and zone will also be marked as the default.
+### Zone
 
-By implementing on the independent realms, zone groups, and zones, Rook supports multiple objects stores in a cluster. The set of users with access to the object store, the metadata, and the data are isolated from other object stores.
+By default, the object store will be created independently from any other object stores and replication to another object store will not be configured. This done by creating a new realm, zone group, and zone all with the name of the new object store.
 
-If desired to configure the object store to replicate from another cluster or zone, the following settings would be specified on a new object store that is *not* the master. (This feature is not yet implemented.)
-- `realm`: If specified, the new zone will be created in the existing realm with that name
-- `group`: If specified, the new zone will be created in the existing zone group with that name
-- `master`: If specified, settings indicate the RGW endpoint where this object store will need to connect to the master zone in order to initialize the replication. The Rook operator will execute `pull` commands for the realm and zone group as necessary.
-```yaml
-  zone:
-    realm: myrealm
-    group: mygroup
-    master:
-      url: https://my-master-zone-gateway:443/
-      accessKey: my-master-zone-access-key
-      secret: my-master-zone-secret
-```
+If desired to configure the object store to replicate and sync data to another cluster, the `zone` section would be required.
 
-Failing over the master could be handled by updating the affected object store CRDs, although more design is needed here.
+This section enables the the object store to join the specified object-store-zone and commit the period after joining it.
```

Comment:
do we require the zone resource to exist before the objectstore is created?

Reply:
Yes.